### PR TITLE
Add initial CONTRIBUTING file

### PR DESCRIPTION
**Pull request by rycee**
Here is a first stab at a "contributing" file…

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+- The entry time stamp should be in ISO-8601 format having "+00:00" as
+  time zone. For example, "2017-09-13T17:10:14+00:00". A suitable time
+  stamp can be produced by the command
+
+      date --iso-8601=second --universal
+
+- The entry condition should be as specific as possible. For example,
+  if you are changing or deprecating a specific option then you could
+  restrict the news to those users that actually use this option.
```
Comment (dermetfan) [inline CONTRIBUTING.md]: `that` -> `who`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+- Wrap the news message so that it will fit in the typical terminal,
+  that is, at most 80 character wide. Ideally a bit less.
+
+- Unlike commit messages, news will be read without any connection to
+  the Home Manager source code. It is therefore important to make the
+  message understandable in isolation and to those that do not have
```
Comment (dermetfan) [inline CONTRIBUTING.md]: `that` -> `who`

**Comment by rycee** [comment]
@dermetfan That's awesome, good catches :-) I've pushed a commit that should fix these problems.

**Comment by rycee** [comment]
I believe all your nice suggestions have been added so I squashed the commits and rebased to master in cab9237d954ebf91b85f0388c35c47cd56674f10. Many thanks for taking the time to read through and commenting!

**Pull request closed without merging**